src/postgres.c

```diff
@@ -72,6 +75,47 @@ _cpycmd(char *host, char *generation)
     return cpycmd;
 }
 
+void
+_commit(Meta *m)
+{
+    PQputCopyEnd((*m)->conn_master, NULL);
+    if ((*m)->conninfo_replica)
+        PQputCopyEnd((*m)->conn_replica, NULL);
+
+    (*m)->count = 0;
+    (*m)->copy  = 0;
+    (*m)->commit_iter = 0;
+}
+
+void *
+_commit_worker(void *meta)
+{
+    Meta *m = (Meta *) meta;
+
+    #if __linux__
+    prctl(PR_SET_NAME, "commit worker");
```

Comment:
It's best to avoid spaces in `argv[0]` when possible (tools parsing `/proc/` have a tendency to choke on unexpected spaces). Unless there're any reasons against it I think it's best to `s/ /_/` to ensure sanity.